doc/news/changes/minor/20171020GiovanniAlzetta

```diff
@@ -0,0 +1,4 @@
+New: Added a method and a enum class to the class BoundingBox. The method is get_neighbor_type which, given another BoundingBox, returns the enum type NeighborType. This enum class describes how the bounding boxes are neighbor.
+Added the function compute_mesh_predicate_bounding_box in grid_tools. This function, given a refinement level, returns a vector of BoundingBoxes for which a predicate is true. There are options to reduce the BoundingBox number with and without changing the space's description given by the union of the BoundingBoxes.
```

Comment:
GridTools::compute_...

Reply:
Note: the reason is that this a) makes it possible to search for the new functions and b) automatically links to them in the doxygen changelog, see http://www.dealii.org/developer/doxygen/deal.II/changes_after_8_5_0.html